src/pages/console/tutorial/bindui.mdx

```diff
@@ -11,7 +11,7 @@ To add a property, choose **Add prop** and set a name for your property. For thi
 
 ## Set child property values
 
-Select a child element such as the "$99 USD" Text element and choose **Set prop**. Here, you are able to set all available properties for that child element. Configure the `label` for the text element and set the value to "Price: $" and then select *Command + Enter* on your keyboard to concatenate another value such as `home.price`.
+Select a child element such as the "$99 USD" Text element and choose **Set prop**. Here, you are able to set all available properties for that child element. Configure the `label` for the text element and set the value to "Price: $" and then select *Command/Ctrl + Enter* on your keyboard to concatenate another value such as `home.price`.
```

Comment:
```suggestion
Select a child element such as the "$99 USD" Text element and choose **Set prop**. Here, you are able to set all available properties for that child element. Configure the `label` for the text element and set the value to "Price: $" and then select `command + return` (or `ctrl + Enter` for Windows) on your keyboard to concatenate another value such as `home.price`.
```

small suggestion to port italics to inline code block. In the future it would be nice to programmatically display different shortcuts based on the viewer's operating system.